.github/workflows/todo.yml

```diff
@@ -19,4 +20,25 @@ jobs:
       run: docker run --env-file .env.test travis_todo todo_app/test/test_appTest.py
     - name: Run end-to-end test
       # You'll need to adjust this to also pass in the relevant environment variables
-      run: docker run --env TOKEN --env KEY --env  SECRET_KEY travis_todo todo_app/test/test_appUiTest.py
\ No newline at end of file
+      run: docker run --env TOKEN --env KEY --env  SECRET_KEY travis_todo todo_app/test/test_appUiTest.py
+
+  deployingtoheroku:
+    name: heroku
+    runs-on: ubuntu-latest
+    # Only deploy when the test pass
+    needs: [test]
+    # if: contains('refs/heads/production', github.ref)  
```

Comment:
Don't forget to uncomment this when you merge (and change `production` -> `master`, unless you're planning to make a production branch)